internal/core/animations.rs

```diff
@@ -295,8 +294,6 @@ pub fn easing_curve(curve: &EasingCurve, value: f32) -> f32 {
                 ctrl2: (*c, *d).into(),
                 to: (1., 1.).into(),
             };
-            #[cfg(feature = "std")]
-            let curve = curve.assume_monotonic();
```

Comment:
I hope this isn’t needed

Reply:
No, this was not needed for my implementation which I copied, iirc.